src/sonic-yang-models/yang-models/sonic-telemetry_client.yang

```diff
@@ -88,20 +88,24 @@ module sonic-telemetry_client {
 
             list TELEMETRY_CLIENT_LIST {
                 ordered-by user;
-                key "prefix";
+                key "prefix name";
 
                 leaf prefix {
                     type string {
-                        pattern 'Subscription_|DestinationGroup_' + '.*';
+                        pattern 'Subscription|DestinationGroup';
```

Comment:
Hello as I can see the prefix is changed from DestinationGroup_ to DestinationGroup and same for DestinationGroup, But the current master branch the  "dialout_client.go" expect the Destination group to be DestinationGroup_.
Example:
 } else if strings.HasPrefix(key, "DestinationGroup_") {
   }
Can you please point out the latest dialout client which can parse DestinationGroup|HS instead of DestinationGroup_HS?